plugins/20_events/hook

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
+source "$(dirname $0)/../common/functions"
+
+[[ ! "$DOKKU_EVENTS" ]] || dokku_log_pluginhook_call "$(basename $0)" "$@"
```

Comment:
Well, actually due ```set -e``` if $DOKKU_EVENTS is unset the hook would return 1, and that would break the plugins chain, wouldn't it?